packages/bundle/src/FullReactWebChat.js

```diff
@@ -31,10 +31,18 @@ class FullReactWebChat extends React.Component {
   }
 
   render() {
-    const { adaptiveCardHostConfig, attachmentMiddleware, styleOptions, styleSet, ...otherProps } = this.props;
+    const {
+      adaptiveCardHostConfig,
+      attachmentMiddleware,
+      styleOptions,
+      styleSet,
+      renderMarkdown,
```

Comment:
Can you check if Prettier is running correctly? This one has sorting problem.

Reply:
Prettier is running, but it's not alphabetizing deconstructed object's properties.